clusterous/cluster.py

```diff
@@ -1483,7 +1483,7 @@ def ls(self, remote_path):
         stdin, stdout, stderr = ssh.exec_command(cmd)
         output_content = stdout.read()
         if 'cannot access' in stderr.read():
-            message = "Error: Folder '{0}' does not exists.".format(remote_path)
+            message = "Folder '{0}' does not exists.".format(remote_path)
```

Comment:
Just realised that these and others should be "..does not exist" not "does not exists." (also note the lack of full stop)